src/components/manage/Contents/Contents.jsx

```diff
@@ -1133,7 +1140,12 @@ class Contents extends Component {
                     <Table.Row>
                       <Table.HeaderCell>
                         <Dropdown
-                          trigger={<Icon name="sort content ascending" />}
+                          trigger={
+                            <Icon
+                              name="sort content ascending"
+                              data-cy="sort"
```

Comment:
@tisto what's the best practice here for marking an element for acceptance testing?

Reply:
@ksuess I remember Timo saying that using `data-` attributes is not the best approach for this, let's see what he says, not sure either that using ids is the best accepted approach either.

Reply:
@ksuess @sneridagh data-cy is recommended by the Cypress docs. I am not sure if I agree personally on this. I tried something similar with Robot Framework and it kind of felt wrong to add something for testing purposes only. We risk to test the wrong this. The right thing to do is always to rely on what an actual user would rely on, e.g. 'click button labled "Save"'. data-cy is kind of the last resort if nothing else works. I guess this is something we have to figure out for Volto in general. Nothing that would prevent this PR to being merged though. I promise to think about it any maybe I find time to write down some Cypress best practices.

Reply:
data-cy: replaced by additional class name.
See "Cypress best practices" #1298